### PR TITLE
update hpa with custom metrics

### PR DESCRIPTION
**Pull request by zouyee**
update hpa with custom metrics which has been updated with v1 version

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4344)

### CODE REVIEW

File: docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This document describes the current state of Horizontal Pod Autoscaling in Kuber
 
 With Horizontal Pod Autoscaling, Kubernetes automatically scales the number of pods
 in a replication controller, deployment or replica set based on observed CPU utilization
-(or, with alpha support, on some other, application-provided metrics).
+(or, with (custom metrics)[https://github.com/kubernetes/community/blob/master/contributors/design-proposals/custom-metrics-api.md] support, on some other, application-provided metrics).
```
Comment (fgrzadkowski) [inline docs/tasks/run-application/horizontal-pod-autoscale.md]: * Parenthesis are wrong - name should be in [] and link in ()
* why do you remove information about this feature being alpha?

Comment (ahmetb) [inline docs/tasks/run-application/horizontal-pod-autoscale.md]: also can use url: `http://git.k8s.io/community/contributors/design-proposals/custom-metrics-api.md`

**Comment by chenopis** [comment]
@zouyee Can you address the comments and merge conflict? thx

File: docs/tasks/run-application/horizontal-pod-autoscale.md (hunk 1 above)
Comment (tengqm) [reply docs/tasks/run-application/horizontal-pod-autoscale.md]: should be http://git.k8s.io/community/contributors/design-proposals/instrumentation/custom-metrics-api.md now

**Comment by zhangxiaoyu-zidif** [comment]
@zouyee  could you rebase this PR and fix it?

**Comment by steveperry-53** [comment]
Ping @zouyee

**Comment by zhangxiaoyu-zidif** [comment]
ping @zouyee 
could you resovle the conflict?

**Comment by steveperry-53** [comment]
In the interest of moving this small change forward, I'm going to create a duplicate.
Duplicate of #6023.

**Pull request closed without merging**